static/docs/user-guide/troubleshooting.md

```diff
@@ -0,0 +1,17 @@
+# Troubleshooting
+
+In this section we cover some of the known issues that DVC user might stumbe
```

Comment:
`know issues` -> replace with something, they are not _issues_, they are expected behavior, right? So, it's a regular workflow you just hit bumps that require a bit more information to deal efficiently with.

Reply:
I'm not sure. See https://github.com/iterative/dvc.org/pull/875#issuecomment-567396421 and https://github.com/iterative/dvc.org/pull/875#issuecomment-567710978.

Reply:
p.s. whatever we decide here should probably also apply to the term "known problem" in line 8 below.

Reply:
@jorgeorpinel see my comment here - https://github.com/iterative/dvc.org/pull/875#issuecomment-567713481 . I don't think we need extra documentation besides GitHub issue tracker for "known issues" and "known problems" (in a sense that they are actual design bugs, bugs and/or are going to be fixed)

Reply:
OK. If you decide this is not a "known issue" though, in fact I think Ivan is also correct in this review. The term "known issue" should definitely not be in the description of a Troubleshooting guide 🙂